sqlite/src/resolve.c

```diff
@@ -514,6 +515,12 @@ static int lookupName(
   if( cnt==0 && zTab==0 ){
     assert( pExpr->op==TK_ID );
     if( ExprHasProperty(pExpr,EP_DblQuoted) ){
+#if defined(SQLITE_BUILDING_FOR_COMDB2)
```

Comment:
We should only be forbidding double-quoted TK_STRINGs (and not TK_IDs).

Reply:
If I recall correctly, the token code here has more to do with the expectation than reality.  I think it's basically just the difference between single and double quotes, as far as the tokenizer is concerned.

Reply:
Ok, while looking at this patch, I wrongly assumed that we haven't yet looked up for a matching column name at this point. I'm ok with this change.